.circleci/config.yml

```diff
@@ -41,10 +41,9 @@ jobs:
       - image: circleci/node:10
     steps:
       - checkout
-      - restore_cache:
```

Comment:
hm.. why no longer the cache?

Reply:
Good point!

So if you look at the file you can see it was currently *only restoring the cache* and nothing else.

My hope was that I could be able to reuse the cache from some previous steps instead of reinstalling the dependencies — but it turns out that since build step and harness are now ran in parallel the thing does not hold anymore.

Reply:
get it now @XVincentX, thanks! So we now can rm https://github.com/stoplightio/prism/blob/a777fb05584788def78904bc86626b007cac0930/.circleci/config.yml#L24 ?

Reply:
Not exactly. The cache can be still be leveraged by the same job that creates it — it simply cannot be used by a different job.

Reply:
ah.. I was looking at the wrong line! My bad!